pkg/specs/githubclient/client_test.go

```diff
@@ -61,23 +61,21 @@ var _ = Describe("GithubClient", func() {
 		Context("With a url prefixed with http(s)", func() {
 			It("should fetch and persist README.md and Chart.yaml", func() {
 				validGitURLWithPrefix := "http://www.github.com/o/r/"
-				mockFs := afero.Afero{Fs: afero.NewMemMapFs()}
 				gitClient := &GithubClient{
 					Client: client,
-					Fs:     mockFs,
 					Logger: log.NewNopLogger(),
 				}
 
 				dest, err := gitClient.GetFiles(context.Background(), validGitURLWithPrefix, constants.HelmChartPath)
```

Comment:
Should these be cleaned up after the fact? I think they should if we're not using a mock fs